python/ray/tune/tuner.py

```diff
@@ -150,11 +150,15 @@ def __init__(
         """Configure and construct a tune run."""
         kwargs = locals().copy()
         self._is_ray_client = ray.util.client.ray.is_connected()
-        if self._is_ray_client and get_air_verbosity() is not None:
-            logger.warning(
-                "Ignoring AIR_VERBOSITY setting, "
-                "as it doesn't support ray client mode yet."
-            )
+        if self._is_ray_client:
+            _run_config = run_config or RunConfig()
+            if get_air_verbosity(_run_config.verbose) is not None:
+                logger.info(
+                    "[output] This will use the legacy output and progress reporter, "
+                    "as Ray client is not supported by the new engine. "
+                    "For more information, please see "
+                    "https://docs.ray.io/en/master/ray-air/experimental-features.html"
```

Comment:
Assuming you intentionally didn't use a relative path here?

Reply:
Yes, it's rendered in the console on runtime (not in the docs)